packages/react-server-cli/README.md

```diff
@@ -0,0 +1,123 @@
+# react-server-cli
+
+A simple command line app that will compile a routes file for the client and start up express. To use:
+
+1. `npm install --save react-server-cli react-server`
+2. Add `./node_modules/react-server-cli/bin/react-server-cli` as the value for `scripts.start` in package.json.
+3. `npm start` from the directory that has your routes.js file.
+
+## Routes Format
+
+Note that the routes file needs to be in a bit different format than what we have used in the past in `react-server`. Rather than `routes.route.page` being a function, it needs to be a path to a file that exports a page class. Middleware also needs to be an array of file paths. For example:
```

Comment:
Totally; it's almost all in compileClient.js. 

In a general sense, I feel very unsure about the question of how many different packages with public APIs this project should be. It feels too monolithic to me right now, especially munging code compilation and server setup. But I'll admit that I don't know right now what the right divisions are.